### ISSUE TITLE
Security link missing from Workspace Settings at certain browser width

### ISSUE THREAD

**Issue by wengole** [ploneintranet/ploneintranet.theme#116]
This issue causes robot tests to fail as the default size window that is opened triggers the bug.

When the window width is approximately less than 942px and greater than 722px, the Security link in the Workspace Settings tile is invisible/missing

![Screenshot of issue](http://i.imgur.com/WD2dJud.png)

**Comment by adrianwhite** [comment]
@cornae do we have a solution for when nav.tabs element becomes to narrow for the content?

**Comment by cornae** [comment]
Nope. But I had a Pattern in mind for that :). /me smiles at @jcbrand …
